neuralprophet/time_dataset.py

```diff
@@ -383,7 +385,7 @@ def _stride_lagged_features(df_col_name, feature_dims):
             tabularized_input_shapes_str += ("    {} {} \n").format(key, value.shape)
     log.debug("Tabularized inputs shapes: \n{}".format(tabularized_input_shapes_str))
 
-    return inputs, targets, config_missing.drop_missing
+    return inputs, targets, config_missing.drop_missing, n_forecasts
```

Comment:
why do we need to return n_forecasts here?

Reply:
I found a solution without returning n_forecasts. pls see PR #747 as this PR will be closed